### ISSUE TITLE
Tree = false not working

### ISSUE THREAD

**Issue by Bizkaitarra** [artgris/FileManagerBundle#20]
I can't make work tree=false configuration.

I first confirgure it in .yaml file and don't work and now I make it using service and it also doesn't work...

In YAML:

artgris_file_manager:
  web_dir: public                 # set your public Directory (not required, default value: web)
  conf:
    default:
      dir: "../public/uploads"
      tree: false

When I make it in service in the service return:

 return ['dir' => $folder, 'tree' => False];

**Comment by artgris** [comment]
https://github.com/artgris/FileManagerBundle/blob/master/Resources/doc/book/3-access-file-manager.md, 

Possible values: 0, 1

**Comment by Bizkaitarra** [comment]
I have now tried with 0 and doesn't work. 

I have made tree diferent tests:
1) Try to use GET parameter in URL, it works
2) Try to use return ['dir' => $folder, 'tree' => 0]; in my service. It does not work, ignores.
3) Try to use yaml configuration, it throws error:
Invalid type for path "artgris_file_manager.conf.otro.tree". Expected boolean, but got integer.
I tried then to write false in tree yaml value and ignore it.
